notebooks/HASP/FluxScaleTutorial/FluxScaleTutorial.ipynb

```diff
@@ -0,0 +1,767 @@
+{
```

Comment:
<p>It seems that step 5 is missing in the notebook. You may want to remove it from the table of contents.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/172/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>